### PR TITLE
add: `komorebi-fork-deb`

### PR DESCRIPTION
**Pull request by 0x262D**

### CODE REVIEW

**Comment by Mar0xy** [comment]
Missing addition to packagelist.
Please add the package name in the packagelist file in the correct alphabetical spot.

**Comment by 0x262D** [comment]
@Mar0xy check pls

File: packages/komorebi-fork-deb/komorebi-fork-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name="komorebi-fork-deb"
+gives="komorebi"
+replace="${gives}"
+version="2.2.1"
+url="https://github.com/Komorebi-Fork/${gives}/releases/download/v${version}/${gives}_${version}-1_amd64.deb"
+description="A beautiful and customizable wallpaper manager for Linux"
+hash="86ef373fb2fb645cdf72af192916dbe278d886283e1ee534843b6d259f3c08ab"
+maintainer="0x262D <pacstall@copypaste.anonaddy.me>"
+repology=("project: komorebi-fork")
+
```
Comment (Henryws) [inline packages/komorebi-fork-deb/komorebi-fork-deb.pacscript:10]: This line isn't needed.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/komorebi-fork-deb/komorebi-fork-deb.pacscript (hunk 1 above)
Comment (0x262D) [reply packages/komorebi-fork-deb/komorebi-fork-deb.pacscript]: removed

**Review by Henryws — APPROVED** [review]

**Pull request merged**